File: results-05-12/python/python-00007-benchmark-basic-tool/qwen2-5-coder-32b/editor/final-src-code.py

import os
import json
os.environ["SHINKAI_NODE_LOCATION"] = "http://localhost:9950"
os.environ["BEARER"] = "debug"
os.environ["X_SHINKAI_TOOL_ID"] = "tool-id-debug"
os.environ["X_SHINKAI_APP_ID"] = "tool-app-debug"
os.environ["X_SHINKAI_LLM_PROVIDER"] = "o_qwen2_5_coder_32b"
os.environ["HOME"] = "results/python/python-00007-benchmark-basic-tool/qwen2-5-coder-32b/editor/home"
os.environ["MOUNT"] = "results/python/python-00007-benchmark-basic-tool/qwen2-5-coder-32b/editor/mount"
os.environ["ASSETS"] = "results/python/python-00007-benchmark-basic-tool/qwen2-5-coder-32b/editor/assets"

from typing import Dict, Any
from shinkai_local_tools import shinkai_foobar

class CONFIG:
    pass

class INPUTS:
    pass

class OUTPUT:
    message: str

async def run(config: CONFIG, inputs: INPUTS) -> OUTPUT:
    response = shinkai_foobar("")
    output = OUTPUT()
    output.message = response["message"]
    return output

if __name__ == "__main__":
    import asyncio
    
    config = CONFIG()
    inputs = INPUTS()
    
    
    # Run the async function
    result = asyncio.run(run(config, inputs))
    print(json.dumps(result.__dict__))

File: results-05-12/typescript/typescript-00018-benchmark-basic-tool/qwen2-5-coder-32b/editor/final-src-code.ts

  // These environment variables are required, before any import.
  // Do not remove them, as they set environment variables for the Shinkai Tools.
  Deno.env.set('SHINKAI_NODE_LOCATION', "http://localhost:9950");
  Deno.env.set('BEARER', "debug");
  Deno.env.set('X_SHINKAI_TOOL_ID', "tool-id-debug");
  Deno.env.set('X_SHINKAI_APP_ID', "tool-app-debug");
  Deno.env.set('X_SHINKAI_LLM_PROVIDER', "o_qwen2_5_coder_32b");
  Deno.env.set('HOME', "results/typescript/typescript-00018-benchmark-basic-tool/qwen2-5-coder-32b/editor/home");
  Deno.env.set('MOUNT', "results/typescript/typescript-00018-benchmark-basic-tool/qwen2-5-coder-32b/editor/mount");
  Deno.env.set('ASSETS', "results/typescript/typescript-00018-benchmark-basic-tool/qwen2-5-coder-32b/editor/assets");
  
import { shinkaiFoobar } from './shinkai-local-tools.ts';

type CONFIG = {};
type INPUTS = {};
type OUTPUT = {
  message: string;
};

export async function run(config: CONFIG, inputs: INPUTS): Promise<OUTPUT> {
    const response = await shinkaiFoobar("Sample message for foobar tool");
    return {
        message: response.message
    };
}

  
  // console.log('Running...')
  // console.log('Config: {}')
  // console.log('Inputs: {}')
  
  try {
    const program_result = await run({}, {});
    if (program_result) console.log(JSON.stringify(program_result, null, 2));
    else console.log(program_result);
  } catch (e) {
    console.log('::ERROR::', e);
  }

